consensus/config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"github.com/spacemeshos/go-spacemesh/log"
+	"time"
+)
+
+// Config is the main configuration of the dolev strong parameters
+type Config struct {
+	NodesPerLayer    int32         `mapstructure:"nodes-per-layer"`
+	PhaseTime        time.Duration `mapstructure:"phase-time"`
```

Comment:
You use both "Phase" and "Round" to describe the same term. I think that "round" is  more suitable (based on the original DS paper where "round" is used for this purpose and "phase" is used for something else)